std/stdio.d

```diff
@@ -802,7 +802,7 @@ $(D rawRead) always reads in binary mode on Windows.
  */
     T[] rawRead(T)(T[] buffer)
     {
-        import std.exception : Exception, errnoEnforce;
+        import std.exception : errnoEnforce;
```

Comment:
`Exception` class is defined in `object` module. So it cannot be imported via `std.exception`.